### PR TITLE
Gnosis Safe modules support

### PR DESCRIPTION
**Pull request by aminlatifi**
Added support for gnosis safe modules
Ref L2B-2941

### CODE REVIEW

**Comment by aminlatifi** [comment]
Blocked by https://github.com/l2beat/tools/pull/92

**Review by aminlatifi — COMMENTED** [review]
I get this error on running discovery with lzomnichain/avalanche
```
Error: Ethereum multicall failed for block number:  33438441. Call size was 1.
```
@adamiak @mateuszradomski is it because of my rpc endpoint?!

File: packages/backend/discovery/amarok/ethereum/discovered.json [hunk 1]
```diff
@@ -458,6 +697,104 @@
...
+        "rescuer": "0x0000000000000000000000000000000000000000",
+        "symbol": "USDC",
+        "totalSupply": "23453926256767876",
+        "TRANSFER_WITH_AUTHORIZATION_TYPEHASH": "0x7c7c6cdb67a18743f49ec6fa9b35f50d52ed05cbed4cc592e13b44501c1a2267",
+        "version": "2"
+      },
+      "errors": {
+        "admin": "Multicall failed",
+        "implementation": "Multicall failed"
+      }
```
[the remark this answers is not in the document]
Comment (michalsidzej) [reply packages/backend/discovery/amarok/ethereum/discovered.json]: [d55094e](https://github.com/l2beat/l2beat/pull/2151/commits/d55094eadee18fef4945f273aa7f945b21cbd70d)

File: packages/backend/discovery/across-v2/ethereum/discovered.json [hunk 2]
```diff
@@ -217,6 +387,52 @@
...
+        "rescuer": "0x0000000000000000000000000000000000000000",
+        "symbol": "USDC",
+        "totalSupply": "23192661498917963",
+        "TRANSFER_WITH_AUTHORIZATION_TYPEHASH": "0x7c7c6cdb67a18743f49ec6fa9b35f50d52ed05cbed4cc592e13b44501c1a2267",
+        "version": "2"
+      },
+      "errors": {
+        "admin": "Multicall failed",
+        "implementation": "Multicall failed"
+      }
```
[the remark this answers is not in the document]
Comment (michalsidzej) [reply packages/backend/discovery/across-v2/ethereum/discovered.json]: [d55094e](https://github.com/l2beat/l2beat/pull/2151/commits/d55094eadee18fef4945f273aa7f945b21cbd70d)

**Review by lucadonnoh — APPROVED** [review]

**Comment by aminlatifi** [comment]
Resolves L2B-2941

**Comment by aminlatifi** [comment]
Resolves L2B-2971

**Pull request merged**